loadtest/control/simplecontroller/controller.go

```diff
@@ -158,18 +154,33 @@ func (c *SimpleController) Run() {
 			waitAfter: 1000,
 		},
 		{
-			run:       c.logout,
-			waitAfter: 1000,
+			run:          c.logout,
+			waitAfter:    1000,
+			runFrequency: 20,
+		},
+		{
+			run:          c.login,
+			waitAfter:    1000,
+			runFrequency: 20,
 		},
 	}
 
 	c.status <- control.UserStatus{ControllerId: c.id, User: c.user, Info: "user started", Code: control.USER_STATUS_STARTED}
 
 	defer c.sendStopStatus()
 
+	c.status <- c.signUp()
+	c.status <- c.login()
+
+	cycleCount := 0 // keeps a track of how many times the entire cycle of actions have been completed.
 	for {
 		for i := 0; i < len(actions); i++ {
-			c.status <- actions[i].run()
+			if actions[i].runFrequency == 0 ||
+				cycleCount%actions[i].runFrequency == 0 {
```

Comment:
I guess calling it *frequency* is a bit counter-intuitive since the greater the value is the less frequently we run the given action. But as long as we make it clear from the docs it shouldn't be a huge problem.

Regarding this check, if we keep this format I don't think 0 is a valid value and I expect a frequency of 1 to mean we run an action on every cycle iteration.

Also, I am wondering if we may want to add a check for `cycleCount == 0` to avoid executing everything on the first iteration which might slightly skew the tests.

Reply:
Is there a better word for "once every x iterations" ? :)

So you want runFrequency of 1 to be set on all actions ? I thought we discussed it would be a non-zero check.

Reply:
> Is there a better word for "once every x iterations" ? :)

Not sure, but I think it's fine as long as we make the behaviour clear. It was more of an afterthought, not a request for change.

> So you want runFrequency of 1 to be set on all actions ? 

I think it should be set to 1 for those actions we want to run on each iteration which is the most part of them at this time.

> I thought we discussed it would be a non-zero check.

Not sure I understood. What I meant above is that on the first iteration when `cycleCount` is 0 we end up running all the actions, regardless of their frequency since the modulo result will be 0.  
I am wondering if this is something we want. Another option to avoid the check is to just start with a `cycleCount` equal to 1. I'd like your thought on this.

Reply:
Ah ok, gotcha. Yes, setting 1 for all other cases is fine I think.